sdgym/sdgym_result_explorer/result_handler.py

```diff
@@ -32,6 +37,106 @@ def load_synthetic_data(self, file_path):
         """Load synthetic data from a file."""
         pass
 
+    @abstractmethod
+    def _load_yaml_file(self, folder_name, file_name):
+        """Load a YAML file from the results folder."""
+        pass
+
+    def _compute_wins(self, result):
+        synthesizers = result['Synthesizer'].unique()
+        datasets = result['Dataset'].unique()
+        result['Win'] = 0
+        for dataset in datasets:
+            score_baseline = result.loc[
+                (result['Synthesizer'] == SYNTHESIZER_BASELINE) & (result['Dataset'] == dataset)
+            ]['Quality_Score'].to_numpy()
+            if score_baseline.size == 0:
+                continue
```

Comment:
I think maybe we should only include datasets that all synthesizers ran on. we can discuss this offline